packages/default-theme/helpers/pages.js

```diff
@@ -0,0 +1,11 @@
+const PAGE_CHECKOUT = '/checkout'
```

Comment:
please do everywhere
```suggestion
export const PAGE_CHECKOUT = '/checkout'
```
and remove single export below